Week-1/Exercise1.1.md

```diff
@@ -0,0 +1,156 @@
+# What is a browser?
```

Comment:
Where is the lucid chart , theory is ok but it needs to be pictorial format

Reply:
Added the image which explains the workflow for web browser.